package.json

```diff
@@ -27,13 +27,12 @@
     "vue-router": "^4.5.1"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^8.31.1",
```

Comment:
based on https://typescript-eslint.io/packages/typescript-eslint/#migrating-from-legacy-eslintrc-configs, @typescript-eslint/eslint-plugin and @typescript-eslint/parser should not be needed separately, typescript-eslint re-exports them